### ISSUE TITLE
inculdeInResultSet for ManagedRelationship

### ISSUE THREAD

**Issue by KRIOFT** [stablekernel/aqueduct#169]
Hi

I have following code
```dart
var query = new Query<City>()
      ..matchOn.country.includeInResultSet = true;

return new Response.ok(await query.fetchOne());
```

it returns me

``` {"error":"Attempting to access matcher on RelationshipInverse country on _City. Assign this value to whereRelatedByValue instead."}```

Off topic: I think is a good feature to can filter in multiple levels like
```dart
var query = new Query<Route>()
      ..matchOn.startCity.country = whereRelatedByValue(1);
// or
var query = new Query<Route>()
      ..matchOn.end.country.name = whereEqualTo('Anglia');

    return new Response.ok(await query.fetchOne());
```

**Comment by joeconwaystk** [comment]
Agreed, and this will make its way into the ORM soon. We just finished up an OAuth2.0 refactor, now working on deployment tools and next up is this kind of stuff for the ORM.

**Comment by joeconwaystk** [comment]
This is now resolved in 2.0.
